src/constants.ts

```diff
@@ -43,6 +34,8 @@ export const PENDING_TXS_KEY = "pendingTxs";
 
 export const POLICY_CHECK_KEY = "policyCheck";
```

Comment:
Does this initial policy check still make sense as it is now?

<img width="551" alt="image" src="https://user-images.githubusercontent.com/15896177/227907952-0a090ca8-1042-4721-aab9-c2688d217006.png">

If it remains, shouldn't it also link to the terms and conditions, privacy policy, and/or risk disclosure URLs?